### ISSUE TITLE
[Bug]: ServerSummary in PersonaBar Does Not Disappear

### ISSUE THREAD

**Issue by mitchelsellers** [dnnsoftware/Dnn.Platform#5771]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What happened?

If you mouse over the DNN logo and see the summary information, it does not disappear on mouse-out.,

### Steps to reproduce?

1. Login as an admin
2. Open the persona bar
3. Mouse over the DNN icon

### Current Behavior

When you move your mouse away from the menu items, 

### Expected Behavior

It should hide after no longer active.

### Affected Versions

9.12.0 (latest release)

### What browsers are you seeing the problem on?

Chrome, Microsoft Edge

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by valadas** [comment]
May be caused by https://bugs.chromium.org/p/chromium/issues/detail?id=133182

**Comment by david-poindexter** [comment]
> May be caused by https://bugs.chromium.org/p/chromium/issues/detail?id=133182

I don't think so - this bug was reported in 2012 and doesn't appear to be something recent.  😉

**Comment by meffj00** [comment]
Dit probleem is van de laatste tijd en doet zich ook voor in versie 9.11.0 met een Google Chrome browser.
